app/javascript/components/contributing/ContributorsList.tsx

```diff
@@ -39,14 +40,19 @@ export default function ContributorsList({
   tracks: readonly Track[]
 }): JSX.Element {
   const { request, setPage, setQuery } = useList(initialRequest)
-  const { status, resolvedData, latestData, isFetching, error } =
-    usePaginatedRequestQuery<PaginatedResult<readonly Contributor[]>>(
-      ['contributors-list', request.endpoint, request.query],
-      {
-        ...request,
-        options: { ...request.options },
-      }
-    )
+  const {
+    status,
+    data: resolvedData,
+    isFetching,
+    error,
+  } = usePaginatedRequestQuery<PaginatedResult<readonly Contributor[]>>(
+    ['contributors-list', request.endpoint, request.query],
+    {
+      ...request,
+      options: { ...request.options },
```

Comment:
Can't this just be:

```suggestion
      ...request
```

Reply:
Yes, this should be simply `request`. I expanded the options object, but ended up not needing any of that.